src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -0,0 +1,2552 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// #define USING_HASH_SET
```

Comment:
Unfortunately my history is incomplete for when that change was entered in 2008.  All of the methods that use that define were introduced at the same time as the define in my history.  It's currently unset on ProjectK and desktop.  I'll remove this.